Dockerfile

```diff
@@ -31,10 +31,13 @@ ENV JWT_SECRET=$JWT_SECRET
 ARG TEST_CURRENT_DATE
 ENV TEST_CURRENT_DATE=$TEST_CURRENT_DATE
 
+ARG API_URL
+ENV API_URL=$API_URL
+
 ENV NODE_ENV=production
 ENV NEXT_TELEMETRY_DISABLED=1
 
-RUN yarn build
+RUN yarn build && yarn --production
```

Comment:
might be better if the build is done outside of the docker build so the image is prod only

Reply:
you mean 2-stage build ?

Reply:
I mean we do on gitlab jobs we yarn and yarn build then we export the result of the build in artefacts to be used in the docker build where we "yarn --production" and copy the build artefacts.